src/modules/video_player/index.js

```diff
@@ -17,6 +18,22 @@ function stepPlaybackSpeed(faster) {
     currentPlayer.player.setPlaybackRate(rates[idx]);
 }
 
+function watchPlayerRecommendationVodsAutoplay() {
+    const player = twitch.getCurrentPlayer().player;
```

Comment:
getCurrentPlayer can return null/undefined .. you should not grab player off of it

Reply:
Is there any recommended way of getting player or should i just guard for it?